components/rating-toast/toast.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/main.css">
+    <link rel="stylesheet" href="/doc.css">
+    <link rel="stylesheet" href="/components/rating-toast/toast.css">
+    <script src="https://kit.fontawesome.com/9c98d67da5.js" crossorigin="anonymous"></script>
+    <title>Avatar</title>
+</head>
+
+<body>
+
+    <nav>
+        <logo class="logo">BuildUI</logo>
+        <ul class="nav-links">
+            <li><a href="#">HOME</a></li>
+            <li><a href="#">DOCUMENTATION</a></li>
+        </ul>
+    </nav>
+
+    <input type="checkbox" id="check">
+    <label for="check"><i class="fas fa-bars" id="btn"></i><i class="fas fa-times" id="cancel"></i></label>
+
+    <div class="side-drawer">
+        <ul>
+            <header>Components</header>
+            <li><a href="/components/html/alert.html">Alert</a></li>
+            <li><a href="/components/html/avatar.html">Avatar</a></li>
+            <li><a href="/components/html/badge.html">Badge</a></li>
+            <li><a href="/components/html/button.html">Button</a></li>
+            <li><a href="/components/html/card.html">Card</a></li>
+            <li><a href="/components/html/image.html">Image</a></li>
+            <li><a href="/components/html/input.html">Input</a></li>
+            <li><a href="/components/html/text-ut.html">Text Utilities</a></li>
+            <li><a href="/components/html/list.html">Lists</a></li>
+            <li><a href="/components/html/nav.html">Navigation-Desktop</a></li>
+            <li><a href="/components/html/modal.html">Modal</a></li>
+            <li><a href="/components/html/rating.html">Rating</a></li>
+            <li><a href="#">Toast/Snackbar</a></li>
+            <li><a href="/components/html/card.html">Simplified Grid</a></li>
+        </ul>
+    </div>
+
+    <div class="Component-container">
+
+      
+            <!---------Toast componenet-------->
+            <div class="text pb2 " id="toast">
+                <h2 class="pb1 h2 comp-title">Toast/Snackbar</h2><br>
+                <p class="pb2">Toasts are lightweight notifications designed to mimic the push notifications that have been popularized by mobile and desktop operating systems. They’re built with flexbox, so they’re easy to align and position.</p>
+                <h3>Example</h3>
+            </div>
+            <main class="main-toast">
+                <div class="toast-container">
+                    <div class="items">can't send photo.Retry in 5 seconds. </div>
+                    <div class="toast-btns">
```

Comment:
Maintain consistent naming of classes with component first and character next.
Example :
instead of toast-btn name it as btn-toast

Reply:
I have updated it,thnks for the review